components/net/hsts.rs

```diff
@@ -132,3 +148,7 @@ pub fn secure_url(url: &Url) -> Url {
         url.clone()
     }
 }
+
+fn reg_host<'a>(url: &'a str) -> String {
```

Comment:
Thanks, I had to add `let host = entry.host.clone();` to solve borrowing issue is this fine?